server/index.js

```diff
@@ -30,6 +30,14 @@ app.get('/api/syllabus/:courseNumber', (req, res) => {
   });
 });
 
+app.get('/api/svg/:svgName', (req, res) => {
+  res.send(JSON.stringify(svgs[req.params.svgName]));
+});
+
+app.get('/api/svgs', (req, res) => {
+  res.send(JSON.stringify(svgs));

```

Comment:
I see that you opened up a route for SVGs, but I didn't see any require or import for the svgs on top of this file so I'm wondering where the "svgs" that the respond is sending back come from

Reply:
@phucciTr to be perfectly honest, I'm not sure! This was already set up in the images service. Maybe it doesn't need to specify a file since they're in the same directory?